kibana/config.sls

```diff
@@ -3,6 +3,15 @@
 
 {% from "kibana/map.jinja" import kibana with context %}
 
+{% if kibana.configfile == '/etc/kibana/kibana.yml' %}
```

Comment:
If you did `makedirs: True` on the state below this, you won't need this new block. Also, this formula already uses a different config directory. So, I don't think you should check against a config different in a different path.

Reply:
My thoughts behind the if clause are that the config file paths are different for version 5 and 4.

> equal or greater 5 /etc/kibana/
> less than 5 /opt/kibana/

therefore I needed to check if it's version 5 and that is done via the configfile path. As someone has to set it in their pillar, else the config file will be deployed to 

> /opt/kibana/config

due to defaults.yaml.

As of second thoughts, I would prefer to write it in another way and have the lines mixed. I'll look into it. I will add this commit here.

Reply:
Then why not `{% if kibana.version.startswith('5') %}`?

Reply:
I also find it odd that the config directory is not created by the package.

Reply:
That's it! `kibana-pkg` should run before `kibana-config`. You can do this with `require` or `require_in`.